crates/zksync/core/src/vm/db.rs

```diff
@@ -87,42 +103,42 @@ where
 
     /// Create a new instance of [ZKEVMData] with system contracts.
     pub fn new_with_system_contracts(ecx: &'a mut EvmContext<DB>, chain_id: L2ChainId) -> Self {
-        let contracts = anvil_zksync_core::deps::system_contracts::get_deployed_contracts(
-            &anvil_zksync_config::types::SystemContractsOptions::BuiltInWithoutSecurity,
-            false,
-        );
+        let contracts = &CACHED_SYSTEM_CONTRACTS;
```

Comment:
redundant variable

Reply:
Wanted to preserve the original way the code was read, but I will inline it then